src/Lib/PHPManagerConfiguration.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace PHPManager\PHPManager\Lib;
+
+final readonly class PHPManagerConfiguration
+{
+    function __construct(public string $phpManagerDirectory = '/.php-manager', public string $distDirectory = '/dist')
```

Comment:
I think the `/` should be the responsibility of the caller, as if you not provide a path as a prefix, I don't think it's good to try to use the system root